gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/medication/MedicationRequestMapper.java

```diff
@@ -61,7 +60,7 @@ private Stream<DomainResource> mapMedicationStatement(RCMRMT030101UKEhrExtract e
 
         var context = encounters.stream()
             .filter(encounter1 -> encounter1.getId().equals(ehrComposition.getId().getRoot())).findFirst();
-        var authoredOn = getAuthoredOn(medicationStatement.getAvailabilityTime(), ehrExtract, ehrComposition);
```

Comment:
So the idea behind this ticket was to remove the fallback, but it appears you've removed the primary time field. Am I confused?